demo/demo_quantized_simulation_letters.py

```diff
@@ -92,6 +92,7 @@ def load_mesh(fn, scale, offset):
     dt_scale=1,
     quant=True,
     use_g2p2g=True,
+    use_bls=False,
```

Comment:
Why do we modify this?

Reply:
It seems that there's some bug with bls which prevents the script from running properly. We're still debugging and will turn on this option again once we fix the problem.